### ISSUE TITLE
Enforce column_limit on strings and comments?

### ISSUE THREAD

**Issue by chrisspen** [google/yapf#755]
Is there any way to enforce the column_limit setting on strings and comments?

I'm finding yapf leaves strings and comments exceeding the column_limit untouched, when I would expect it to wrap them. Is this by design or is there another setting to enforce the column_limit on lines globally?

**Comment by gwelymernans** [comment]
Yeah, this is by design. YAPF won't modify any of the tokens in the token stream, if at all possible. This is to reduce bugs and ensure that the original semantics of the program are preserved.

I would be willing to accept a patch to do this, but it would not be on by default.

**Comment by kamahen** [comment]
By contrast, "black" changes backslashes to parenthesized form and adds trailing commas to parameter lists. It might be nice to have that option in yapf, but I like yapf's philosophy.

**Comment by dslim23** [comment]
agreed this would be nice to have
